### ISSUE TITLE
saas-18: All settings menus broken.

### ISSUE THREAD

**Issue by nhomar** [odoo/odoo#19553]
Impacted versions: **saas-18**

```
Error:
Uncaught TypeError: widget.renderWithLabel is not a function

http://localhost:8069/web/content/486-920c66d/web.assets_backend.js:1337
Traceback:
TypeError: widget.renderWithLabel is not a function
    at http://localhost:8069/web/content/486-920c66d/web.assets_backend.js:1337:52
    at Function._.each._.forEach (http://localhost:8069/web/content/485-86f0652/web.assets_common.js:12:558)
    at Class._updateView [as _super] (http://localhost:8069/web/content/486-920c66d/web.assets_backend.js:1336:157)
    at Class._updateView (http://localhost:8069/web/content/486-920c66d/web.assets_backend.js:2322:20)
    at Class._updateView (http://localhost:8069/web/content/485-86f0652/web.assets_common.js:3814:371)
    at http://localhost:8069/web/content/486-920c66d/web.assets_backend.js:1335:1479
    at http://localhost:8069/web/content/485-86f0652/web.assets_common.js:802:681
    at fire (http://localhost:8069/web/content/485-86f0652/web.assets_common.js:796:299)
    at Object.add [as done] (http://localhost:8069/web/content/485-86f0652/web.assets_common.js:797:467)
    at Array.<anonymous> (http://localhost:8069/web/content/485-86f0652/web.assets_common.js:802:649)
```

Steps to reproduce:

Go to settings.

![image](https://user-images.githubusercontent.com/338634/30583422-a2f18dde-9cec-11e7-9cad-8fb65a954d97.png)

**Comment by gdgellatly** [comment]
@nhomar master as well. just didn't report as settings has been broken in different ways for weeks now there.

**Comment by sle-odoo** [comment]
fixed by rev ae14f93464ac9878180b50f262a84804538fb6f3 in enterprise

thanks for the report
